xbmc/platform/darwin/tvos/Assets.xcassets/LaunchImage.launchimage/Contents.json

```diff
@@ -3,15 +3,15 @@
     {
       "orientation" : "landscape",
       "idiom" : "tv",
-      "filename" : "splash.jpg",
+      "filename" : "splashscreen.png",
       "extent" : "full-screen",
       "minimum-system-version" : "9.0",
       "scale" : "1x"
     },
     {
       "orientation" : "landscape",
       "idiom" : "tv",
-      "filename" : "splash@2x.jpg",
+      "filename" : "splashscreen@2x.png",
```

Comment:
unrelated to your change, but there was no, and now also is no `@2x` splash image. So shouldn't this entire config block get removed, or is this file autogenerated during packaging? @fuzzard?

Reply:
There's a symlink on the folder